code/lab1-tdd-triangle/src/main/java/com/lahtachev/triangle/Triangle.java

```diff
@@ -0,0 +1,118 @@
+package com.lahtachev.triangle;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.Math.*;
+
+public class Triangle {
+    private final Point point1;
+    private final Point point2;
+    private final Point point3;
+    private final int sideNumber = 3;
+
+    public Triangle(final Point point1, final Point point2, final Point point3) {
+        if (point1.equals(point2) || point2.equals(point3)) {
+            throw new IllegalArgumentException("Points should not be in one place");
```

Comment:
Maybe will be better do this logic in separate method, not in constructor.

Reply:
Done